suite-native/module-accounts-import/src/screens/XpubScanScreen.tsx

```diff
@@ -65,7 +65,10 @@ const submitButtonStyle = prepareNativeStyle(utils => ({
 const DEFAULT_CURRENCY_SYMBOL = 'btc';
 // Slip 0014 - https://github.com/satoshilabs/slips/blob/master/slip-0014.md#bitcoin-segwit--p2wpkh--bip84
 const BTC_HARD_CODED_XPUB =
-    'xpub6BiVtCpG9fQPxnPmHXG8PhtzQdWC2Su4qWu6XW9tpWFYhxydCLJGrWBJZ5H6qTAHdPQ7pQhtpjiYZVZARo14qHiay2fvrX996oEP42u8wZy';
+    'zpub6rjNNddoAVvuYaD6WPdxiqFEToQHgrERjWMg7kM9gGGk6rhPMWNEmL5X745FGqBq8Wp136LfA3A7UjRGEYdJrf8dUfshzNrb5rvaryNfVJf';
```

Comment:
Why do we change this xpub in nearly every PR? :D

Reply:
This Xpub is from @sime and actually has some some btc so it will be good for testing. It's some company testing xpub apparently.

Reply:
I don't think this needs to be in the repo, but if it's convenient, go for it.